400-claims.md

```diff
@@ -193,18 +193,29 @@ To satisfy these requirements, implementations of a CNAB package manager are exp
 
 Compliant CNAB implementations MUST conform to this section.
 
+### Environment Variables
+
 The claim is produced outside of the CNAB package. The following claim data is injected
 into the invocation container at runtime:
 
 - `$CNAB_INSTALLATION_NAME`: The value of `claim.name`.
 - `$CNAB_BUNDLE_NAME`: The name of the present bundle.
 - `$CNAB_ACTION`: The action to be performed (install, upgrade, ...)
 - `$CNAB_REVISION`: The ULID for the present release revision. (On upgrade, this is the _new_ revision)
+- `$CNAB_CLAIMS_VERSION`: The version of this specification (currently `CNAB-Claims-1.0-WD`)
 
 > Credential data, which is also injected into the invocation image, is _not_ managed by the claim system. Rules for injecting credentials are found in [the bundle runtime definition](103-bundle-runtime.md).
 
 Parameters declared with an `env` key in the `destination` object MUST have their values injected as environment variables according to the name specified. Likewise, files MUST be injected if `path` is set on `destination`.
 
+### Files
+
+The invocation image may benefit from accessing the claim. Consequently, a claim MUST be attached to the invocation image when the invocation image is started.
+
+The claim MUST be mounted at the path `/cnab/claim.json` inside of the bundle. The version of this claim that is to be mounted is the _version prior to this operation beginning_. In other words, when a bundle is installed, it creates the original installation claim. On the first upgrade, the claim describe the _installation_ is located at `/cnab/claim.json`. This allows the invocation image to inspect the former state and compare it to the desired new state.
```

Comment:
Perhaps reword `the claim describe` to `the claim describing` or `the claim that describes`?